app/views/help/index.html.erb

```diff
@@ -0,0 +1,63 @@
+<div>
+  <h2 class="govuk-heading-l">Help</h2>
+
+  <p class="govuk-body">
+    <h2 class="govuk-heading-s">Issues</h2>
+    <details class="govuk-details">
+      <summary class="govuk-details__summary">
+        <span class="govuk-details__summary-text">
+          My end user is having problem connecting to GovWifi
+        </span>
+      </summary>
+      <div class="govuk-details__text">
+        <p class="govuk-body">
+          If an end user is having trouble with GovWifi, we recommend directing them to <%= link_to 'our guidance on common issues', @end_user_docs_link %>.
+        </p>
+        <p class="govuk-body">
+          If this doesn't resolve the issue, we recommend they contact their local support - we don't provide end user support directly
+        </p>
+        <p class="govuk-body">
+          If you're seeing systemic issues with GovWifi - lots of your end users are having trouble, but your access point is connecting to our RADIUS servers correctly - then check our <%= link_to 'guidance for resolving access point issues', @organisation_docs_link %>.
+        </p>
+      </div>
+    </details>
+  </p>
+
+  <p class="govuk-body">
+    <h2 class="govuk-heading-s">Before you contact us</h2>
+
+    <ul class="govuk-list govuk-list--bullet">
+      <li><%= link_to 'Check servers status', status_index_path %></li>
+      <li><%= link_to 'Browse technical documentation', @organisation_docs_link %></li>
+    </ul>
+  <p>
+<div>
+
+<%= form_tag('/help') do %>
+  <div>
```

Comment:
Could probably do with a little margin above here to separate the "Contact Us" title from the content above. Perhaps a `govuk-grid-row`? https://design-system.service.gov.uk/styles/layout/